### DISCUSSION TITLE
Line items description - Asian chars

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#16894] [Ignored/Incomplete]
Line items description by default should support any language but it not. Worth to fix it in core

**Comment by mariuszkrzaczkowski** [comment]
is this a bug report?  If so, this is not the section and there is no description of how to call the error

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/issues/13908
